### ISSUE TITLE
offset support

### ISSUE THREAD

**Issue by dalishi** [esmini/esmini#189]
Hi is there a plan to add "offset" support to RoadManager? Then the x/y coordinates calculated by Position::setLanePos() will be automatically added by this offset? Thanks.

OpenDRIVE                       
|-header                         
| |-geoReference             
| |-offset

**Comment by eknabevcc** [comment]
@ralurral Do you have an answer to this question?

**Comment by ralurral** [comment]
Yes, there is a plan to add it, we don't know yet how will be implemented.

**Comment by eknabevcc** [comment]
Ok good. We can look into it together at next planning. Let's keep this issue open until then.
